File: 2.0/java/calls/getCallRecordings.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CallRecording;

import java.util.List;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        List<CallRecording> recs = client.callsApi().getCallRecordings(379506003L, "items(callId,campaignId,lengthInBytes,mp3Url)");
    }
}

File: 2.0/csharp/calls/getCallRecordings.cs
using System.Collections.Generic;
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        // limit returned fields if needed
        IList<CallRecording> recordings = client.CallsApi.GetCallRecordings(379506003, "items(callId,campaignId,lengthInBytes,mp3Url)");
    }
}
